lms/static/js/utils/navigation.js

```diff
@@ -0,0 +1,73 @@
+var edx = edx || {},
+
+    Navigation = (function() {
+
+        var navigation = {
+
+            init: function() {
+
+                if ($('#accordion').length) {
```

Comment:
Previous code that this was abstracted from assumed only one accordion so I kept that through this code. This is specifically related to the course navigation of which only one is present at a time.